lib/app.tsx

```diff
@@ -166,18 +169,18 @@ export const App = connect(
         .on('update', debounce(this.props.loadTags, 200))
         .on('remove', this.props.loadTags);
 
-      const {
-        actions: { setConnectionStatus },
-      } = this.props;
-
       this.props.client
         .on('authorized', this.onAuthChanged)
         .on('unauthorized', this.onAuthChanged)
         .on('message', setLastSyncedTime)
         .on('message', this.syncActivityHooks)
         .on('send', this.syncActivityHooks)
-        .on('connect', () => setConnectionStatus({ isOffline: false }))
-        .on('disconnect', () => setConnectionStatus({ isOffline: true }));
+        .on('connect', () => {
+          this.props.setSimperiumConnectionStatus(true);
+        })
```

Comment:
fun fact! we don't need the curlies

```ts
.on('connect', () => this.props.setSimperiumConnectionStatus(true))
```

Reply:
ohh, true. I think I had console.logs in there for testing.